ledger-obsolete/src/main/kotlin/net/corda/v5/ledger/obsolete/services/vault/NamedQueryFilter.kt

```diff
@@ -0,0 +1,6 @@
+package net.corda.v5.ledger.obsolete.services.vault
+
+import net.corda.v5.base.annotations.CordaSerializable
+
+@CordaSerializable
+interface NamedQueryFilter
```

Comment:
FYI @blsemo, @vlajos 
We removed post filter/post processing from the public persistence API. But as legacy ledger is still referencing this, I moved the objects here so it continues to compile.
Please let me know if this is an issue.